src/traces/sankey/render.js

```diff
@@ -701,13 +732,17 @@ module.exports = function(gd, svg, calcData, layout, callbacks) {
         })
         .style('stroke-width', function(d) {
             return salientEnough(d) ? d.linkLineWidth : 1;
-        });
+        })
+        .attr('d', linkPath());
```

Comment:
Did moving this `attr()` call down to the merge selection fix any bugs?

Reply:
Yes it did. After a grouping operation, some links stay in the figure needs (they don't `enter` the scene) but their `d` attribute still need to be updated.

Reply:
Have you tried adding a test for this?

Reply:
> Have you tried adding a test for this?

No, I haven't. What specifically would you test?

Reply:
that an updated node gets the correct `'d'` string.

Reply:
Done in ffbafa7